### DISCUSSION TITLE
CONCOURSE_ENABLE_P2P_VOLUME_STREAMING leads to p2p errors

### DISCUSSION THREAD

**Discussion by alex1x** [concourse/concourse#7388] [Help & Support]
Setting `CONCOURSE_ENABLE_P2P_VOLUME_STREAMING` to true led to certain resource checks failing with `run check: find or create container on worker epsteindidntki: failed to get p2p url: 500`. 

Worker version: `2.3` (as reported by `fly workers`)
Web version: `7.4.0` (as reported by UI)

Setting the parameter to false fixes the issue.

I don't know if it's actually related but I tried setting `CONCOURSE_BAGGAGECLAIM_P2P_INTERFACE_NAME_PATTERN` to match my network interface but that made no difference. Any ideas?

**Comment by evanchaoli** [comment]
Most likely the problem is about that Concourse couldn't get an interface for p2p streaming. Can you show `ifconfig -a` of your worker and what is `CONCOURSE_BAGGAGECLAIM_P2P_INTERFACE_NAME_PATTERN` you set?

Please note that, p2p streaming requires all workers are on the same network and they can reach to each other.

**Reply by vbrunko-voc** [reply]
Yes, it solved for me!
![image](https://github.com/concourse/concourse/assets/67569220/d2aaf708-883f-4e8d-8bce-bd23d2baf42f)
